libfuzzer-dotnet.cc

```diff
@@ -7,6 +7,9 @@
 #include "unistd.h"
 #include <sys/shm.h>
 
+#define _STR(x) #x
```

Comment:
This is just a CPP helper to let us define the `.._FD` constants as ints, but also use them as _string_ constants when passing them via env vars.